dist/hijax.js

```diff
@@ -69,25 +69,22 @@
 
     // Get a particular response header value by key
     // Extracted from jQuery 2.1.1
+    // Changed to grab content-type only (we only use content-type, no need to set everything) from XHR object
     Hijacker.prototype.getResponseHeader = function(xhr, key) {
```

Comment:
@henryh15 if we're going to change the purpose of the function, we should rename it and remove the unused variables.